### ISSUE TITLE
Building Extension Fails with unknown property 'archiveBaseName'

### ISSUE THREAD

**Issue by VGKintsugi** [NationalSecurityAgency/ghidra#3313]
**Describe the bug**
I am failing to build my Ghidra extension. I am using the command: `gradle -PGHIDRA_INSTALL_DIR=/home/user/Desktop/ghidra_10.0.1_PUBLIC/`

This fails with:
```
FAILURE: Build failed with an exception.

* Where:
Script '/home/user/Desktop/ghidra_10.0.1_PUBLIC/support/buildExtension.gradle' line: 80

* What went wrong:
A problem occurred evaluating script.
> Could not set unknown property 'archiveBaseName' for task ':buildExtension' of type org.gradle.api.tasks.bundling.Zip.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

```

**To Reproduce**
1. git clone https://github.com/VGKintsugi/Ghidra-SegaSaturn-Loader
2. Extract Ghidra 10.0.1 from releases
3. Build extension with: gradle -PGHIDRA_INSTALL_DIR=/home/user/Desktop/ghidra_10.0.1_PUBLIC/`

**Fix**
Looking at `/home/user/Desktop/ghidra_10.0.1_PUBLIC/support/buildExtension.gradle` line 80 and 81 it appears that that solution is to prepend the word "def" to the lines. So the lines should be:

```	
def archiveBaseName = "${ZIP_NAME_PREFIX}_${project.name}"
def archiveExtension = 'zip'
```
instead of:
```
archiveBaseName = "${ZIP_NAME_PREFIX}_${project.name}"
archiveExtension = 'zip'
```
Making this change I am able to build my extension.

**Comment by ryanmkurtz** [comment]
What version of Gradle are you using?

**Comment by ryanmkurtz** [comment]
I was not able to reproduce on both Gradle 6.8.3 and Gradle 7.1.

**Comment by VGKintsugi** [comment]
I am using Gradle 4.4.1 which is what was available in apt in Linux Mint 20.2. I can try updating Gradle but it would have to be outside of apt.

**Comment by ryanmkurtz** [comment]
Ghidra 10.x requires Gradle 6 or 7. We have checks for this when building Ghidra. I'll look into adding the check to building extensions too.

**Comment by astrelsky** [comment]
> I am using Gradle 4.4.1 which is what was available in apt in Linux Mint 20.2. I can try updating Gradle but it would have to be outside of apt. 
> 

That is ancient. Have you run `sudo apt update` or is it because mint uses some ancient java 5 or something?

**Comment by topia** [comment]
I don't know why they didn't update it, but the latest ubuntu contains Gradle 4.4.1 too. https://packages.ubuntu.com/impish/gradle
I suggest using the gradle wrapper.
